src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -179,7 +179,11 @@ public void load(CoreExtension.Context context) {
                             .name("Repository Slug for the Gitlab (Server or Cloud) instance")
                             .description("The repository slug can be either in the form of user/repo or it can be the Project ID")
                             .type(PropertyType.STRING)
-                            .build()
+                            .build(),
+                            
+                    PropertyDefinition.builder(GitlabServerPullRequestDecorator.PULLREQUEST_COMPACT_COMMENTS_ENABLED).category(PULL_REQUEST_CATEGORY_LABEL).subCategory(GITLAB_INTEGRATION_SUBCATEGORY_LABEL)
```

Comment:
this would be for me also a property that can be used by the bitbucket provider too. I see you hardcoded it currenlty there to false.

Ideally the property is available for all providers.